GW2EIEvtcParser/EncounterLogic/Fractals/ShatteredObservatory/Skorvald.cs

```diff
@@ -73,12 +73,32 @@ internal override List<PhaseData> GetPhases(ParsedEvtcLog log, bool requirePhase
                     phase.Name = "Split " + (i) / 2;
                     var ids = new List<int>
                     {
-                        (int)ArcDPSEnums.TrashID.FluxAnomaly4,
-                        (int)ArcDPSEnums.TrashID.FluxAnomaly3,
-                        (int)ArcDPSEnums.TrashID.FluxAnomaly2,
                         (int)ArcDPSEnums.TrashID.FluxAnomaly1,
+                        (int)ArcDPSEnums.TrashID.FluxAnomaly2,
+                        (int)ArcDPSEnums.TrashID.FluxAnomaly3,
+                        (int)ArcDPSEnums.TrashID.FluxAnomaly4,
                     };
                     AddTargetsToPhaseAndFit(phase, ids, log);
+
+                    // add anomaly numbers
+                    int offset = 4 * (i / 2 - 1);
```

Comment:
Having the overrides be in GetPhases is a little bit problematic as that means that as long as GetPhases method hasn't been called, the name of the targets would be wrong.
Something like Dictionary<TrashID, int> { {FluxAnomaly1, 1} ... {FluxAnomaly4, 1}} where the count is fetched and incremented in EIEvtcParse would be better.
Same for the Artsariiv clones, a dictionary based on CloneLocations and have your utility method return the matched <string, float, float > maybe?

Reply:
I see. For Skorvald that could work. For Artsariiv the N, E, S, W clones only appear in the second split. I would still prefer to name the first occurrence of those "Artsariiv N 2" after the phase they occur in, if that's fine?